circle.yml

```diff
@@ -4,11 +4,15 @@ machine:
 dependencies:
   override:
     - eval "$(curl -sL https://apt.vapor.sh)"
-    - sudo apt-get install vapor cmysql
+    - sudo apt-get install swift vapor cmysql
     - sudo chmod -R a+rx /usr/
 test:
   override:
     - swift build
     - swift build -c release
     - swift test
-    
\ No newline at end of file
+    - sudo apt-get remove swift
+    - sudo apt-get install swift-beta
```

Comment:
actually this is out of date! I should probably flag this on the Vapor repo, the `swift` package is now Swift 4 - if you do `sudo apt-get install swift=3.1` that should install the old version